lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -1453,6 +1453,8 @@ SwiftASTContext *TypeSystemSwiftTypeRef::GetSwiftASTContext() const {
         *const_cast<TypeSystemSwiftTypeRef *>(this));
     m_swift_ast_context =
         llvm::dyn_cast_or_null<SwiftASTContext>(m_swift_ast_context_sp.get());
+    if (m_swift_ast_context && m_swift_ast_context_triple)
```

Comment:
If we wanted to get rid of the optional that would be: `!m_swift_ast_context_triple.str().empty()`